Dockerfile.zephyr

```
@@ -0,0 +1,69 @@
+# Install Renode for S32K388 simulation (Fase 6+)
+RUN wget -q https://github.com/renode/renode/releases/download/v${RENODE_VERSION}/renode-${RENODE_VERSION}.linux-portable.tar.gz \
+    && mkdir -p /opt/renode \
+    && tar xzf renode-${RENODE_VERSION}.linux-portable.tar.gz -C /opt/renode --strip-components=1 \
+    && rm renode-${RENODE_VERSION}.linux-portable.tar.gz \
+    && ln -s /opt/renode/renode /usr/local/bin/renode
+
+# Create workspace
+WORKDIR /workspace
+
+# Initialize Zephyr workspace with west
+RUN west init /workspace/zephyrproject && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)